qiskit/quantum_info/synthesis/two_qubit_decompose.py

```diff
@@ -38,7 +38,7 @@
 from qiskit.circuit.quantumcircuit import QuantumCircuit, Gate
 from qiskit.circuit.library.standard_gates import CXGate, RXGate, RYGate, RZGate
 from qiskit.exceptions import QiskitError
-from qiskit.quantum_info.operators import Operator
+from qiskit.quantum_info.operators.operator import Operator
```

Comment:
Is this change necessary?  These kind of hyper-specific imports are part of why we sometimes have deprecation problems with names imported into various modules.  If `qiskit.quantum_info.operators` is a complete package that exports `Operator`, we should be able to import it from there.

(Usually we'd be able to import it from `qiskit.quantum_info`, but since we're still constructing the `quantum_info` package here in this file, we do sometimes need more specificity.  I also know this package is overly specific in other imports, my comment's just because there's a change here.)

Reply:
Reverted in ce4d070fb.